internal/tiltfile/k8s.go

```diff
@@ -338,10 +338,12 @@ func starlarkValuesToJSONPaths(values []starlark.Value) ([]k8s.JSONPath, error)
 }
 
 func (s *tiltfileState) k8sImageJsonPath(thread *starlark.Thread, fn *starlark.Builtin, args starlark.Tuple, kwargs []starlark.Tuple) (starlark.Value, error) {
-	var kind, name, namespace string
+	var group, version, kind, name, namespace string
 	var imageJSONPath starlark.Value
 	if err := starlark.UnpackArgs(fn.Name(), args, kwargs,
 		"path", &imageJSONPath,
+		"group?", &group,
```

Comment:
have we double-checked this syntax with somebody external?

usually public-facing k8s yaml uses apiVersion instead "{group}/{kind}", e.g., "core/v1". 

for users who have never hacked on kubernetes internals, i wouldn't expect them to be familiar with "group"